src/router.js

```diff
@@ -19,6 +19,11 @@ export default new Router({
       // which is lazy-loaded when the route is visited.
       component: () =>
         import(/* webpackChunkName: "about" */ "./views/About.vue")
+    },
+    {
+      path: "*",
```

Comment:
```suggestion
    {
      path: '/404',
      name: 'NotFound',
      component: () => import('./views/NotFound.vue')
    },
    {
      path: '*',
      redirect: {name: 'NotFound'}
    }
```
This will allow the 404 view to have a defined path while support redirects.